src/logger/loggerApi.c

```diff
@@ -1085,25 +1085,41 @@ static const jsmntok_t * setPidExtendedField(const jsmntok_t *valueTok, const ch
 	return valueTok + 1;
 }
 
+/* Max number of PIDs that can be written in the setOBD2Config message */
+#define MAX_OBD2_MESSAGE_PIDS 10
```

Comment:
OK to put the #define close to where it would be used, or is it better form to put it at the top? was conflicted about this.